scripts/setup-data-dir.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root"
+   exit 1
+fi
+
+datadir="${1}"
+
+if [[ -z "${1}" ]]; then
+    datadir="/var/lib/rkt"
+fi
+
+md() {
+    local dir="${1}"
+    local mode="${2}"
+
+    if [[ -e "${dir}" ]]; then
+	chmod "${mode}" "${dir}"
+    else
+	mkdir --mode="${mode}" "${dir}"
+    fi
+    chgrp rkt "${dir}"
```

Comment:
I don't think so, the whole point is to prepare the directory structure for rkt group, otherwise rkt will happily create required directories on first run.

`set -e` is global here.